app/instance-initializers/ember-i18n.js

```diff
@@ -1,12 +1,24 @@
 import Ember from "ember";
+import Stream from 'ember-i18n/stream';
 import legacyHelper from "ember-i18n/legacy-helper";
 import ENV from '../config/environment';
 
 export default {
   name: 'ember-i18n',
 
-  initialize: function(instance) {
+  initialize({ container }) {
```

Comment:
FYI - accessing `instance.container` will trigger a deprecation in 2.1.  Moving this destructuring down into the `if` below will ensure it is only ran on pre-1.13 (where `instance.container` is fine).